clients/rust/src/client_inference_params.rs

```diff
@@ -14,7 +14,7 @@ use uuid::Uuid;
 // This is a copy-paste of the `Params` struct from `tensorzero_internal::endpoints::inference::Params`.
 // with just the `credentials` field adjusted to allow serialization.
 /// The expected payload is a JSON object with the following fields:
-#[derive(Debug, Serialize, Default)]
+#[derive(Clone, Debug, Serialize, Default)]
```

Comment:
why does this need Clone?

Reply:
One of the e2e tests makes the same request twice to a flaky dummy model.  I think this is also generally useful for clients making repeated inferences.